website/docs/docs/dbt-versions/release-notes/05-Aug-2023/ide-v1.2.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Update: Cloud IDE v1.2 with dbt-server"
+description: "August 2023: Cloud IDE now uses dbt-server to provide more reliable service and dbt Core feature parity, including support for commands like `dbt list`."
+sidebar_label: "Update: Cloud IDE v1.2"
+tags: [Aug-2023, IDE]
+date: 2023-08-03
+sidebar_position: 8
+---
+
+We're excited to announce that we replaced the backend service that powers the Cloud IDE with a more reliable server, dbt-server. This significant update follows the rebuild of the IDE frontend last year. We're committed to improving our IDE to provide you with a better experience.
+
+The Cloud IDE previously used dbt-rpc, an outdated service that was unable to keep up with the changes from dbt-core. The dbt-rpc integration used legacy dbt-core entry points and logging systems, causing it to be sluggish, brittle, and not well tested. At a high cost of maintenance, the Core team has been working around this outdated technology to avoid breaking it, preventing them from developing with velocity and confidence.
```

Comment:
```suggestion
Previously, the Cloud IDE used dbt-rpc, an outdated service that was unable to stay up-to-date with changes from dbt-core. The dbt-rpc integration used legacy dbt-core entry points and logging systems, causing it to be sluggish, brittle, and not well tested. At a high cost of maintenance, the Core team has been working around this outdated technology to avoid breaking it, preventing them from developing with velocity and confidence.
```